client/components/app-promo/index.jsx

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import assign from 'lodash/object/assign';
+import classNames from 'classnames';
+import noop from 'lodash/utility/noop';
+import Gridicon from 'components/gridicon';
+
+import analytics from 'analytics';
+import store from 'store';
+
+export default React.createClass( {
+
+	displayName: 'AppPromo',
+
```

Comment:
Looks like there is a prop of `location` being passed in, there should be a `propTypes` declaration up here, and `location` might be set to `isRequired` to properly record the track event.